modules/tfm/zephyr/CMakeLists.txt

```diff
@@ -178,3 +170,54 @@ set_property(TARGET zephyr_property_target
 )
 
 zephyr_include_directories(${NRF_DIR}/include/tfm)
+
+# Default values from config_base.h in TF-M.
```

Comment:
```suggestion
# Default values from config_base.h in TF-M. Overridable by setting a CMake Cache variable of the same name that can shadow these variables.
```

Assuming this is true, and we don't have another short-term strategy for overriding.

Not having a strategy for configuring these values would be a problem.

Reply:
It's not true, and the strategy is to properly integrate with TF-Ms Kconfig system, this is just temporary to bring back the existing configurations.
I try to avoid adding features / capabilies in this PR, I just want to get back the existing functionality with the updated version.

Reply:
Discussed offline.

This should be addressed before release at least.

There used to be a supported documented way to configure these options and this change is a regression that makes TF-M less configurable.

https://docs.zephyrproject.org/3.2.0/services/tfm/build.html#custom-cmake-arguments